package.json

```diff
@@ -11,7 +11,8 @@
     "test": "echo 'Add tests please :)'",
     "gql:compile": "graphql-codegen",
     "gql:watch": "graphql-codegen -w",
-    "type-check": "tsc -p ./tsconfig.json"
+    "type-check": "tsc -p ./tsconfig.json",
+    "clean": "rimraf .next"
```

Comment:
nice! I did something similar in dms-ui, and this here is the end result of my troubleshooting
https://github.com/overture-stack/dms-ui/blob/develop/package.json#L8

Reply:
good shout. added.

Reply:
oh I meant "and 'dev'" too, for good measure